src/components/TableView.ts

```diff
@@ -0,0 +1,37 @@
+import * as vscode from "vscode";
+import fs = require("fs");
+import mustache = require("mustache");
+
+export interface TableResult {
+  Head: Array<string>;
+  Rows: Array<Array<string>>;
+}
+
+export class TableView {
+  public static show(
+    context: vscode.ExtensionContext,
+    result: TableResult,
+    title: string
+  ) {
+    const panel = vscode.window.createWebviewPanel(
+      "InfluxDB",
+      title,
+      vscode.ViewColumn.Two,
+      {
+        retainContextWhenHidden: true
+      }
+    );
+
+    panel.webview.html = TableView.getWebviewContent(context, result);
+  }
+
+  public static getWebviewContent(
+    context: vscode.ExtensionContext,
+    result: TableResult
+  ): string {
+    let temp = String(
+      fs.readFileSync(context.asAbsolutePath("templates/table.mst"))
```

Comment:
you probably don't want this in the function call, I would move this to a static variables as it will stop the event loop every time you call this function.